lib/spack/spack/util/web.py

```diff
@@ -86,25 +98,39 @@ def __init__(self, *args, **kwargs):
             super(NonDaemonPool, self).__init__(*args, **kwargs)
 
 
-def _read_from_url(url, accept_content_type=None):
+__UNABLE_TO_VERIFY_SSL = (
+        lambda pyver: (
```

Comment:
why is there a `lambda` in here?  Why not just make this a clearly written function?